### DISCUSSION TITLE
post: Create a Vue Library by Vite and Publish to NPM

### DISCUSSION THREAD

**Discussion by giscus** [wghglory/my-website#53] [Announcements]
# post: Create a Vue Library by Vite and Publish to NPM

Guanghui Wang Website

https://derekw.vercel.app/posts/how-to-create-a-vue-library

**Comment by jimwilson94** [comment]
Guanghui, would you please add tests and storybook?

**Reply by wghglory** [reply]
Sure thing, will do that.
